nestjs/src/registry/registry.controller.ts

```diff
@@ -29,4 +30,21 @@ export class RegistryController {
       userId: user.id,
     });
   }
+
+  @Get('mentors')
+  @ApiOperation({ operationId: 'getMentorRegistries' })
+  @RequiredRoles([Role.Admin, CourseRole.Manager, CourseRole.Supervisor])
+  @ApiOkResponse({ type: [MentorRegistryDto] })
+  public async getMentorRegistries(@Req() req: CurrentRequest) {
+    if (req.user.isAdmin) {
+      const data = await this.mentorsService.findAllMentorRegistries();
+      return data.map(el => new MentorRegistryDto(el));
```

Comment:
Let's not use generic stuff like `el` here, it would be slightlty better if we could see what variable contains by its name :)

Reply:
I'll fix it when I add pagination here